frontend/views/pages/Payments.vue

```diff
@@ -458,7 +458,7 @@ export default ({
 
       const matchesMethodFilter = methodFilterVal === 'all' || acceptedMethods.includes(methodFilterVal)
       const matchesSearchQuery = searchQuery === '' ||
-        `${amount}${username.toUpperCase()}${displayName.toUpperCase()}`.indexOf(searchQuery.toUpperCase()) !== -1
+        `${amount}${displayName.toUpperCase()}`.indexOf(searchQuery.toUpperCase()) !== -1
```

Comment:
it appears that `username` was replaced with `displayName` following it at some point but `username` hasn't been removed, causing this run-time error.

![image](https://github.com/user-attachments/assets/1dfa2cc8-7cb7-428c-b6b7-a1bde4ba0c8e)